data/isdv4-49c8.tablet

```diff
@@ -0,0 +1,19 @@
+# Dell Inspiron 7425 14 2-in-1
+# Sensor Type: AES (
+# Features: Touch (Intergrated), Tilt
+#
```

Comment:
Please add the comments listing + linking to the sysinfo, like e.g. in de8d5883b61346d0a997c347640498e310d30231, thanks

Reply:
Before doing the update, will it be correct to list something like _Dell Inspiron 14 7425 2-in-1 (Wacom HID 49C8)_ as name of device?

Reply:
everything else in the file is good as it is, the `Name` field should match the kernel's name (doubly so since the same device may be used in other devices too).